crates/bevy_ui/src/lib.rs

```diff
@@ -12,6 +12,8 @@ pub mod ui_material;
 pub mod update;
 pub mod widget;
 
+#[cfg(feature = "debug")]
```

Comment:
I think we should rename this feature to something like `debug_layout`, just to reduce ambiguity.